src/WellIO.jl

```diff
@@ -1,5 +1,26 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+
+function load(fname::File{format"LAS"})
+    s = open(fname) do file
+        read(file,String)
+    end
```

Comment:
You are incorrectly using the identifier `fname` to represent an argument of type `File`. Please rename it to `file`. Given a file, you can open it to get a file handle, not a `file`. Please rename `file` to `f`. I would expect that all these lines could be replaced by `read(file, String)` after the names are changed, but I am not sure.